docs/projects/rot-13/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ROT-13 in Every Language
+layout: default
+date: 2018-11-20
+last-modified: 2018-11-20
+featured-image:
+tags: [rot-13]
+authors:
+---
+
+ROT13 is a letter substitution cipher where every letter is replaced by the
```

Comment:
There are a handful of consistency issues here with the naming of ROT13/ROT-13. I don't mind which way you go, but it should probably be consistent throughout this document.

Reply:
I decided to go with ROT-13. I think I got them all. Let me know if I missed one.